miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebViewClient.kt

```diff
@@ -26,6 +26,13 @@ internal class MiniAppWebViewClient(
         return response
     }
 
+    override fun onPageFinished(view: WebView?, url: String?) {
+        if (url?.startsWith(miniAppScheme.miniAppCustomDomain) == true && url.contains(miniAppScheme.parentUrlExtension))
+            view?.clearHistory()
```

Comment:
@khairul-alam-licon . I don't think we should clear the history. It should stay the same. Please check

Reply:
Yeah, I agree. This is a solution particularly for this issue, but clearing history will affects other area. I'm looking for any other solution. Thank you.